cmake/PALTools.cmake

```diff
@@ -16,11 +16,11 @@
 # Discover all the platforms that are available
 
 set(LY_PAL_TOOLS_DEFINES)
-file(GLOB pal_tools_files "cmake/Platform/*/PALTools_*.cmake")
+file(GLOB pal_tools_files cmake/Platform/*/PALTools_*.cmake)
```

Comment:
We should keep these paths in double quotes in case the use's directory structure contains spaces with them.

Reply:
done